### DISCUSSION TITLE
Shouldn't the "Build a blog" tutorial be adjusted to the newer Content Collections API?

### DISCUSSION THREAD

**Discussion by ElianCodes** [withastro/docs#2663] [Ideas]
So, the Build a blog tutorial still uses `Astro.glob('../pages/posts/*.md')` . Wouldn't it be a better idea and intro to Astro to convert this to the newer Astro Content Collections API?

Willing to work on this!

**Comment by delucis** [comment]
Thanks for raising this @ElianCodes! We actually do have a guide to adding content collections to the tutorial which we’re going to link to at the end: https://github.com/withastro/docs/pull/2668

For now we decided to skip adding this to the tutorial directly, because the config can add an extra layer of complexity for people new to Astro that isn’t strictly necessary — basically we want to avoid the impression that that complexity is _required_ for working with content.
